mypy/tvar_scope.py

```diff
@@ -68,6 +68,7 @@ def bind(self, name: str, tvar_expr: TypeVarExpr) -> TypeVarDef:
 
     def get_binding(self, item: Union[str, SymbolTableNode]) -> Optional[TypeVarDef]:
         fullname = item.fullname if isinstance(item, SymbolTableNode) else item
+        assert fullname is not None
```

Comment:
Yes, I have added a line in ``semanal.analyze_lvalue``, so that class members (i.e. ``MDEF`` nodes) always get their ``fullname`` generated (I don't know why it was not like this before, maybe just an omission). Now this should never be ``None``.